# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/html/HTMLFormControlAccessible.h
# Commit: 5d76535b47fc
# Full Hash: 5d76535b47fced08e46753a0262211732fb26417
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P7: Move and rename GetPositionAndSizeInternal to Accessible. r=Jamie
#   
#   Now that the AccGroupInfo getter is in Accessible, the position and size
#   getter could go there too.
#   
# ==============================================================================

diff -r fca1ff7eabf4 -r 5d76535b47fc accessible/html/HTMLFormControlAccessible.h
--- a/accessible/html/HTMLFormControlAccessible.h	Tue Jan 04 21:01:36 2022 +0000
+++ b/accessible/html/HTMLFormControlAccessible.h	Tue Jan 04 21:01:37 2022 +0000
@@ -31,10 +31,12 @@
 
   // LocalAccessible
   virtual uint64_t NativeState() const override;
-  virtual void GetPositionAndSizeInternal(int32_t* aPosInSet,
-                                          int32_t* aSetSize) override;
   virtual Relation RelationByType(RelationType aType) const override;
 
+ protected:
+  virtual void GetPositionAndSetSize(int32_t* aPosInSet,
+                                     int32_t* aSetSize) override;
+
  private:
   Relation ComputeGroupAttributes(int32_t* aPosInSet, int32_t* aSetSize) const;
 };